.changeset/flat-pears-hug.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/cloudflare': minor
+---
+
+Deprecates adapter options
```

Comment:
Does it deprecate ALL the options? Just some of them? I think it would be helpful to be explicit here:

e.g.

> Deprecates the following adapter options:

> Deprecates all of the current adapter options except:

And for the sake of making it easier for people to figure out, so that they don't have to jump into the code to learn what happened, I would honestly put your code comments here in this changelog. That's what this space is for, to tell people what *changed* and what their options are!